MmSupervisorPkg/Plugin/PostBuildScanMmStandalone_plug_in.json

```diff
@@ -0,0 +1,12 @@
+## @file

```

Comment:
you should probably change this to yaml which is allowed to have comments.  JSON isn't technically allowed to have comments so tools show errors.  

Also this looks like copy/paste from a different plugin.  Probably needs to be updated.